test/setup.ts

```diff
@@ -5,7 +5,7 @@ const lt = new LocalTableland({ silent: true });
 
 before(async function () {
   this.timeout(15000);
-  lt.start();
+  await lt.start();
   await lt.isReady();
```

Comment:
Not sure how I missed this before, but the linting pointed out that the awaiting start method makes waiting on isReady unnecessary.

Reply:
Does isReady only resolve when start is done?

Reply:
Yep, there were some changes to local-tableland that happened end of last week that make that the case.

Reply:
I think so. Since it's used internally by local-tableland we could keep exposing it as a convenience function, but in the case of these tests it's not needed anymore.

Reply:
Nice, let's drop it from the example here. Ever _simpler_ APIs!